examples/package.yaml

```diff
@@ -0,0 +1,54 @@
+name: example
+version: '1.0'
+maintainer: TVision Insights
+license: AllRightsReserved
```

Comment:
Need a package name, `0.x` version, and license.

Reply:
Right, I'm not going to release this, it's just so I know it compiles when I load in GHCi. I will add a license.